app/services/wordpress_service.rb

```diff
@@ -24,10 +24,6 @@ def blog_post_views(blog_post_id)
     get_response_body("sites/#{site_id}/stats/post/#{blog_post_id}")
   end
 
-  def blog_post(blog_post_id)
-    get_response_body("sites/#{site_id}/posts/#{blog_post_id}")
-  end
-
```

Comment:
I know we are going to stop using it, but I'd keep this method regardless, just in case we (or they) can fix the problem sometime down the line.
Is there any linter forcing you to do it?

Reply:
code climate coverage is failing because is under 99%, I thought it was because of this but it seems not

Reply:
Could you restore it, then?